awscli/customizations/cloudformation/deploy.py

```diff
@@ -169,6 +169,30 @@ class DeployCommand(BasicCommand):
                 ' execute it to implement your changes.'
             )
         },
+        {
+            'name': 'disable-rollback',
+            'required': False,
+            'action': 'store_true',
+            'group_name': 'disable-rollback',
+            'dest': 'disable_rollback',
+            'default': False,
+            'help_text': (
+                'Preserve the state of previously provisioned resources when '
+                'the execute-change-set operation fails.'
+            )
+        },
+        {
+            'name': 'no-disable-rollback',
```

Comment:
Rather than `--no-disable-rollback` it may be clearer to use something like `--enable-rollback`. Thoughts?

Reply:
We are trying to keep the flag aligned across all of our public apis, see below public link as a reference, we use `no-disable-rollback`
https://docs.aws.amazon.com/cli/latest/reference/cloudformation/create-stack.html

Reply:
Fair enough, if there's established precedent that's fine.